packages/framework/aqueduct/src/legacy-container-runtime-factories/README.md

```diff
@@ -0,0 +1,3 @@
+# Legacy container runtime factories
```

Comment:
I'd move it to some distinct directory path / package, i.e. out of aqueduct.

Reply:
Ah, I see, there is also base factory in here. Probably fine to be here, but I fear that over time there would be a bunch of code like that, where ideally we separate not just by having deprecated property, but maybe also by having deprecated in package name?

Reply:
It's no problem -- I'll separate it out to `@fluidframework/deprecated-aqueduct`.  I think it goes away in N+2 anyway.

It's hard to do a general-purpose deprecated package for deprecated stuff across the framework because of our layer checks, I think.  Or else we'd need to make it a big exception to the layer checks.